src/structures/PartialGuild.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+const Base = require('./Base');
+const Permissions = require('../util/Permissions');
+const SnowflakeUtil = require('../util/Snowflake');
+
+/**
+ * Represents a partial guild, received when using {@link GuildManager#fetch} to fetch multiple guilds.
+ * @extends {Base}
+ */
+class PartialGuild extends Base {
```

Comment:
Since this exists, can we not make Guild extend this?

Reply:
Guild doesn't have `.permissions` and also a different value for `.owner`, so that would be a bit problematic there